code/modules/projectiles/guns/ballistic/automatic.dm

```diff
@@ -915,6 +915,8 @@
 	item_state = "slr"
 	force = 20
 	mag_type = /obj/item/ammo_box/magazine/m762
+	extra_damage = 35
```

Comment:
Plus yo ass had to add DAMAGE TO THE GUN, CAUSE IT AIN'T BEEN USED FOR A DECADE PLAYA, 35 DAMAGE AND 0.1 AP.... GOD DAMN....